### PR TITLE
Feat/connect popup r

### LINKED ISSUE #7514: Change copy for T2B1
i. e. 
image

### PR DESCRIPTION
**Pull request by tomasklim**
## Description

- R correctly used in connect-popup

## Related Issue

closes #7514

## Notes

- I am not happy with the style how trezor icon images are used here, but let's leave this for some refactoring, there is also some weird `icomoon` stuff and it can be done together
- Has to be little bit changed when we add `getDeviceDisplayName` method, so that we do not hardcore those names because of `Trezor Model One`

Screenshot 2023-07-17 at 13 20 26

### CODE REVIEW

File: suite-native/module-home/src/components/DashboardHeaderDeviceChip.tsx [hunk 1]
```diff
@@ -27,7 +27,7 @@ export const DashboardHeaderDeviceChip = () => {
                 onPress={() => setIsDeviceModalVisible(true)}
             >
                 <Icon name="trezorT" />
-                <Text>Trezor T</Text>
+                <Text>Trezor Model T</Text>
                 <Box>
                     <Icon name="chevronUp" size="small" />
                     <Icon name="chevronDown" size="small" />
```
Comment (juriczech) [inline suite-native/module-home/src/components/DashboardHeaderDeviceChip.tsx]: This file is unused. Fixed here https://github.com/trezor/trezor-suite/pull/8911

File: packages/connect-popup/src/view/firmwareNotCompatible.ts [hunk 2]
```diff
@@ -22,7 +22,7 @@ export const firmwareNotCompatible = (device: UiRequestUnexpectedDeviceMode['pay
     const confirmButton = view.getElementsByClassName('confirm')[0] as HTMLButtonElement;
     const cancelButton = view.getElementsByClassName('cancel')[0] as HTMLButtonElement;
 
-    fwVersion.innerHTML = `${features.major_version}.${features.minor_version}.${features.patch_version}`;
+    fwVersion.innerHTML = getFirmwareVersion();
```
Comment (matejkriz) [inline packages/connect-popup/src/view/firmwareNotCompatible.ts]: missing device param

File: suite-native/module-home/src/components/DashboardHeaderDeviceChip.tsx (hunk 1 above)
Comment (tomasklim) [reply suite-native/module-home/src/components/DashboardHeaderDeviceChip.tsx]: commit skipped

File: packages/connect-popup/src/view/firmwareNotCompatible.ts (hunk 2 above)
Comment (tomasklim) [reply packages/connect-popup/src/view/firmwareNotCompatible.ts]: fixed, thx

**Review by matejkriz — APPROVED** [review]
It looks good to me, would be nice if @mroz22 double check it

File: packages/connect-popup/src/static/popup.html [hunk 3]
```diff
@@ -1995,7 +1995,7 @@ <h3>New firmware update available</h3>
             <!-- Firmware not supported -->
             <div class="firmware-not-supported">
                 <div>
-                    <h3>Trezor One is not supported</h3>
+                    <h3>Coin is not supported</h3>
```
Comment (mroz22) [inline packages/connect-popup/src/static/popup.html]: this is a courageous change. Wouldn't it be better to text it something like
`Connected Trezor model not supported` 
?

**Review by mroz22 — APPROVED** [review]
lgtm

File: packages/connect-popup/src/static/popup.html (hunk 3 above)
Comment (tomasklim) [reply packages/connect-popup/src/static/popup.html]: The string should never be used. However, makes sense, I will update it immediately

**Pull request merged**